source/dub/generators/generator.d

```diff
@@ -591,6 +591,10 @@ struct GeneratorSettings {
 
 	// only used for generator "build"
 	bool run, force, direct, rdmd, tempBuild, parallelBuild;
+
+	/// Selected dub packages are stored in dub.selections.json
+	saveSelections;
```

Comment:
isn't this missing `bool`?

also for backwards compatibility and API use this should probably be ` = true`

Reply:
and actually how this is used, the comment should probably say something like:

```
/// checks if dub.selections.json is updated for rebuild check if true
```

if I read the code correctly.

Reply:
Fixed